CHANGES.md

```diff
@@ -65,6 +65,8 @@
 
 #### _Black_
 
+- Improve github issue template "running on your machine"'s instructions (#2100)
```

Comment:
I think we can skip the news entry because this change isn't relevant to people looking to upgrade

Reply:
Also, this change isn't going to be in 20.8b0

Reply:
> I think we can skip the news entry because this change isn't relevant to people looking to upgrade

travis state will be red then, but I supposed it's not a matter here, I haven't read contributions guidlines ;)

Reply:
reverted !